ex_standards/ex_http_status.md

```diff
@@ -9,34 +9,34 @@ Some might ok with 2 options, choose one and justify it.
 ----
 
 * I cannot access that web if I'm not logged:
-  *  
+  401 Unauthorized.
 
 * I'm logged but I cannot access that page
-  * 
+  403 forbidden
 
 * The item with that Id was deleted
-  * 
+  200 OK
 
 * I started uploading, but I have to check another link to see if it is there
-  * 
+  206 Partial Content
 
 * That query parameter is wrong
-  * 
+  400 bad request
 
 * There are no resources with that ID
-  * 
+  404 Not found
 
 * My request was good, but it will take time to resolve
-  * 
+  202 Accepted
 
 * The resource was deleted but no extra info was provided
-  * 
+  204 Not content
 
 * Added a product to the collection
-  * 
+  201 Created
 
 Got the info I needed
-  * 
+  200 Ok
 
```

Comment:
a maioría das respostas perfectas, un par de comentarios para opcións mellores ou máis habituais